app/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml.scala

```diff
@@ -112,27 +122,32 @@ object DeclarationXml {
           <GovernmentAgencyGoodsItem>
             <SequenceNumeric>{dec.declarationType.goodsItemNumber}</SequenceNumeric>
             <AdditionalDocument>
```

Comment:
can we instead do something like
`{ dec.documentation.additionalDocuments.map(maybeAdditionalDocument) }`
to omit the additionaldocument elements where nothing has been entered?  might need to tweak the form mapping to achieve this though.

Reply:
This doesn't allow for situation where user has entered a space.